src/uucore/src/lib/features/selinux.rs

```diff
@@ -6,22 +6,34 @@
 use std::path::Path;
 
 use selinux::SecurityContext;
+use thiserror::Error;
```

Comment:
This currently fails with an "unresolved import" when running `cargo test --features=feat_selinux --no-default-features`. I think you have to add `thiserror` to the `selinux` feature in `Cargo.toml`.